src/masoniteorm/models/Model.py

```diff
@@ -590,12 +590,14 @@ def save(self, query=False):
             else:
                 result = self.create(self.__dirty_attributes__, query=query)
             self.observe_events(self, "saved")
+            self.fill(result.__attributes__)
```

Comment:
I don't think it is __attributes__ right? Because the result of the create will contain the `id` ... 🤔

Reply:
shouldn't it be:

```python
self.fill(result)
```

Reply:
When I used the self.fill(result) it raises this in `fill` method:

```python
self.__attributes__.update(attributes)
...
TypeError: getattr(): attribute name must be string
```

If I'm not wrong the type of "result" is a Model ( Category) , then I use the atributes returned by result.